io-engine/src/subsys/config/opts.rs

```diff
@@ -668,7 +668,7 @@ impl Default for IoBufOpts {
     fn default() -> Self {
         Self {
             small_pool_count: try_from_env("IOBUF_SMALL_POOL_COUNT", 8192),
-            large_pool_count: try_from_env("IOBUF_LARGE_POOL_COUNT", 1024),
+            large_pool_count: try_from_env("IOBUF_LARGE_POOL_COUNT", 2047),
```

Comment:
Sorry if it wasn't clear, but the -1 is to be done only for num_shared_buffers. For iobuf pool count it can remain as it is, the spdk defaults are full value too:
> #define IOBUF_DEFAULT_SMALL_POOL_SIZE	8192
#define IOBUF_DEFAULT_LARGE_POOL_SIZE	1024

The transport shared buffers is what's needed to be at -1

> #define SPDK_NVMF_TCP_DEFAULT_NUM_SHARED_BUFFERS 511

Reply:
Right yes, done